client/src/components/QuestionList.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useCallback } from "react";
+
+import { Card, TextField, Stack, Badge } from "@shopify/polaris";
+
+export function QuestionList({ questions, isAdmin }) {
+  const [questionTitle, setQuestionTitle] = useState("");
+  const [rangeMin, setRangeMin] = useState("");
+  const [rangeMax, setRangeMax] = useState("");
+  const [textPrompt, setTextPrompt] = useState("");
+  const [options, setOptions] = useState("");
+  const handleChange = useCallback(newTitle => setQuestionTitle(newTitle), []);
+  const handleMinChange = useCallback(newMin => setRangeMin(newMin), []);
+  const handleMaxChange = useCallback(newMax => setRangeMax(newMax), []);
+  const handleTextChange = useCallback(newText => setTextPrompt(newText), []);
+  const handleOptionChange = useCallback(
+    newOptions => setOptions(newOptions),
+    []
+  );
+
+  return questions && questions.length > 0
+    ? questions.map((question, index) => {
```

Comment:
Turnery operators are head to read we should make this into a if statement latter

Reply:
just use glasses

Reply:
"its an elegant way of making a declarative UI" -ismail syed when debating tobi lutke